package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "loast",
   "description": "CLI for testing Lighthouse APIs using OpenAPI specs",
-  "version": "0.2.0",
+  "version": "0.0.0-development",
```

Comment:
Where did the decision come for this to be the initial version? Is it a "semantic release" standard?

Reply:
Yes, this is what semantic release sets your version to when you initialize it.